### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ens-st3**
## Description
Minor updates:
- replace 2020 with beta
- focus on current features
- remove Custom Download

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1754

## Deployment URL(s)
None

## Views affected
None, github only

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 This repository is an all-new frontend for the Ensembl website, written in React, Rust and WASM. The current public release can be seen at https://beta.ensembl.org. The new site will integrate genomes from across the taxonomic space, including vertebrates, invertebrates, plants, fungi and bacteria.
 
-If you are interested in the latest developments in this project, please join our Slack channel #ensembl2020 or follow our [blog](https://www.ensembl.info).
+If you are interested in the latest developments in this project, please join our Slack channel #ensemblbeta or follow our [blog](https://www.ensembl.info).
```
Comment (ens-st3) [inline README.md:7]: Waiting on systems to rename

File: README.md [hunk 2]
```diff
@@ -32,20 +32,19 @@ The new genome browser uses Rust, WASM and WebGL to provide a fast, smooth inter
...
 
-We are replacing BioMart with a more scalable and performant solution that allows you to quickly configure a custom dataset and download it. (Note that there will also be downloads of individual sequences available through other apps such as Entity Viewer.)
+You can search for gene names from any of the applications above. Global Search, which will allow you to search for gene names and symbols and other identifiers is for future development.
 
-### Global Search
-
-Similar to our current site-wide search, the Global Search will allow you to search for gene names and symbols and other identifiers.
+### BLAST
 
+You can BLAST against a range of databases (proteins, transcripts, and genomic sequence) view the results in graphic and tabular format, and download the data.
```
Comment (azangru) [inline README.md:47]: comma before "view"?

File: README.md [hunk 3]
```diff
@@ -32,20 +32,19 @@ The new genome browser uses Rust, WASM and WebGL to provide a fast, smooth inter
...
 ### Entity Viewer
 
-View detailed information about annotation features: genes, transcripts, proteins, variants, etc.
+View detailed information about annotation features: genes, transcripts and proteins.
 
-### Custom Download
+### Gene name Search
 
-We are replacing BioMart with a more scalable and performant solution that allows you to quickly configure a custom dataset and download it. (Note that there will also be downloads of individual sequences available through other apps such as Entity Viewer.)
+You can search for gene names from any of the applications above. Global Search, which will allow you to search for gene names and symbols and other identifiers is for future development.
```
Comment (azangru) [inline README.md:43]: comma after "other identifiers"?

File: README.md [hunk 4]
```diff
@@ -2,9 +2,9 @@
 
 ## Introduction
 
-This repository is an all-new frontend for the Ensembl website, written in React, Rust and WASM. The current public release can be seen at https://2020.ensembl.org. The new site will integrate genomes from across the taxonomic space, including vertebrates, invertebrates, plants, fungi and bacteria.
+This repository is an all-new frontend for the Ensembl website, written in React, Rust and WASM. The current public release can be seen at https://beta.ensembl.org. The new site will integrate genomes from across the taxonomic space, including vertebrates, invertebrates, plants, fungi and bacteria.
```
Comment (azangru) [inline README.md:5]: Should we mention Rust and WASM anymore in the readme for this repo? All the Rust code has been moved out of it.

**Review by azangru — APPROVED** [review]

**Pull request merged**